docs/specifications/tests/DNSSEC-TP/dnssec11.md

```diff
@@ -211,7 +211,7 @@ No special terminology for this test case.
 
 
 [Argument list]:                              https://github.com/zonemaster/zonemaster-engine/blob/master/docs/logentry_args.md
-[Basic04]:                                    ../Basic-TP/basic014.md
+[Connectivity01]:                             ../Basic-TP/basic014.md
```

Comment:
../Basic-TP/basic014.md -> ../Connectivity-TP/connectivity01.md

Reply:
Thanks for catching it. Funny that I missed it since I did the update of all files with a one-liner. Fixed.